templates/kselftest/generic-arm-uboot-zimage-dtb-ramdisk-kselftest-template.jinja2

```diff
@@ -0,0 +1,54 @@
+{% extends 'base/kernel-ci-base-uboot.jinja2' %}
+{% set nfsrootfs_url = 'https://people.collabora.com/~tomeu/lava-files/stretch-armhf-nfs.tar.gz' %}
```

Comment:
I'm not really sure about having files hosted like this, and I don't really know where the plan for kselftest is going, so maybe talk to @khilman about what you're doing here